core/esmf-aspect-meta-model-python/scripts/download_test_models.py

```diff
@@ -0,0 +1,67 @@
+#  Copyright (c) 2024 Robert Bosch Manufacturing Solutions GmbH
+#
+#  See the AUTHORS file(s) distributed with this work for additional
+#  information regarding authorship.
+#
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+#   SPDX-License-Identifier: MPL-2.0
+
+from os import mkdir, remove
+from os.path import exists, join
+from pathlib import Path
+from zipfile import ZipFile
+
+import requests
+
+
+FOLDER_TO_EXTRACT = "valid"
+TEST_MODELS_PATH = join("tests", "integration", "java_models", "resources")
+VERSION = "2.7.0"
```

Comment:
In `download_samm_cli.py` there is another constant, `SAMM_CLI_VERSION` (which currently points to a different esmf-sdk version!). There should be only one place where the esmf-sdk version is defined.

Reply:
Move all constants to the separate file